documentation/synthetic_tokens/security/bug_bounty.md

```diff
@@ -0,0 +1,78 @@
+# UMA Bug Bounty Program
+
+Security of the platform is our highest priority. 
+All contract code and balances are publicly verifiable, and security researchers are eligible for a bug bounty for reporting undiscovered vulnerabilities.
+
+## Audit
+<!-- TODO: Add details of audit. 
+The contracts have been carefully audited by smart contract security experts [names]. 
+The audit report can be found [here].
+--> 
+
+UMA encourages the community to audit our contracts and security; we also encourage the responsible disclosure of any issues.
+The bug bounty program is intended to recognize the value of working with the community of independent security researchers, and sets out our definition of good faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
+
+## Rewards
+UMA offers substantial rewards for discoveries that can prevent the loss of assets, the freezing of assets, or harm to users. 
+
+Rewards range from $500 to $25,000. 
+All rewards will be paid in ETH, and the amount of compensation will vary depending on Severity. 
+The severity is calculated according to the [OWASP](https://owasp.org/www-project-risk-assessment-framework/) risk rating model based on Impact and Likelihood.
+
+![](severity.png)
+
+## Scope
+<!-- TODO: add version name of DVM 
+The scope of our bug bounty program includes core smart contracts related to release [v1.0] of the DVM and financial templates. 
+-->
+It does not include known issues with the intended behavior. 
+<!-- TODO: create overview document for system
+Please refer to the [readme file] in the repository for an extensive overview of the intended behavior of the system.
+--> 
+
+### In scope:
+* DVM contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/oracle/implementation)
+* Multi-sponsor token contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/financial-templates/implementation)
```

Comment:
Technically we don't want this to be part of the bug bounty until after we get OZ's audit back. 

@mrice32 what do you think of making a note stating that we're accepting bugs for the DVM now, and that we'll be accepting bugs for the NFCT after say May 1?

Reply:
That seems reasonable. We could also just remove this line and add when we're ready to add it to the scope. Either way SGTM.

Reply:
It makes sense to me to remove this line (comment it out) and add it back when we're ready.